trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -9,8 +9,8 @@
         <th scope="col" class="govuk-table__header" aria-sort="ascending">Bank</th>
         <th scope="col" class="govuk-table__header" aria-sort="none">Status</th>
         <th scope="col" class="govuk-table__header" aria-sort="none">Date report received</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Total facilities reported</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Facilities left to reconcile</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Total fees reported</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Reported fees left to reconcile</th>
```

Comment:
Are there any e2e-tests which need to be updated to match the new text in this file?